pkg/etl/Makefile.include

```diff
@@ -0,0 +1,25 @@
+INCLUDES += -I$(PKGDIRBASE)/etl/include
+INCLUDES += -I$(RIOTPKG)/etl/config
+
+# There's nothing to build in this package, it's used as a header only library.
+# So it's declared as a pseudo-module
+PSEUDOMODULES += etl
+
+# Activate the usage of libstdcpp types and features if available
+# This prevents unnecessary reimplementations
+ifneq (,$(filter libstdcpp,$(FEATURES_USED)))
+  CXXEXFLAGS += -DFEATURE_LIBSTDCPP
+endif
+
+# Some boards don't provide NAN or by default don't define limits like UINT8_MAX in CPP context
+# This activates the appropriate workarounds
+ifneq (,$(filter arch_avr8,$(FEATURES_USED)))
+  CXXEXFLAGS += -DNO_CPP_NAN_SUPPORT
+  CXXEXFLAGS += -D__STDC_LIMIT_MACROS
+else ifneq (,$(filter arch_riscv,$(FEATURES_USED)))
+  CXXEXFLAGS += -DNO_CPP_NAN_SUPPORT
+  CXXEXFLAGS += -D__STDC_LIMIT_MACROS
+endif
```

Comment:
These settings are not activated when building with Kconfig.

Do I need to check for the Kconfig specific defines extra? Unfortunately, I haven't really understood how the Kconfig build works and interacts with the Makefiles yet.

```suggestion
# Activate the usage of libstdcpp types and features if available
# This prevents unnecessary reimplementations
ifneq (,$(filter libstdcpp,$(FEATURES_USED)))
  CXXEXFLAGS += -DFEATURE_LIBSTDCPP
# Kconfig alternatives
else ifndef HAS_LIBSTDCPP
  CXXEXFLAGS += -DFEATURE_LIBSTDCPP
endif

# Some boards don't provide NAN or by default don't define limits like UINT8_MAX in CPP context
# This activates the appropriate workarounds
ifneq (,$(filter arch_avr8,$(FEATURES_USED)))
  CXXEXFLAGS += -DNO_CPP_NAN_SUPPORT
  CXXEXFLAGS += -D__STDC_LIMIT_MACROS
else ifneq (,$(filter arch_riscv,$(FEATURES_USED)))
  CXXEXFLAGS += -DNO_CPP_NAN_SUPPORT
  CXXEXFLAGS += -D__STDC_LIMIT_MACROS
# Kconfig alternatives
else ifndef CPU_ARCH_AVR8
  CXXEXFLAGS += -DNO_CPP_NAN_SUPPORT
  CXXEXFLAGS += -D__STDC_LIMIT_MACROS
else ifndef CPU_ARCH_RISCV
  CXXEXFLAGS += -DNO_CPP_NAN_SUPPORT
  CXXEXFLAGS += -D__STDC_LIMIT_MACROS
endif
```

Reply:
Should be fixed by 74b7a26 hopefully.